apiserver/application/application.go

```diff
@@ -57,6 +60,15 @@ type API struct {
 	// should pass a charm.Charm and charm.URL back into
 	// state wherever we pass in a state.Charm currently.
 	stateCharm func(Charm) *state.Charm
+
+	deployApplicationFunc func(backend Backend, args jjj.DeployApplicationParams) error
+}
+
+// DeployApplication is a wrapper around juju.DeployApplication, to
+// match the function signature expected by NewAPI.
+func DeployApplication(backend Backend, args jjj.DeployApplicationParams) error {
```

Comment:
Personally, I'd rather this not be exported.
This is a case where adding an alias to export_test makes sense IMO.